abahachuk/ptushki-frontend
# update: EPMCUXDA-7721 update initial data shape

### What does this PR do?

Updates accordingly to change provided by https://github.com/abahachuk/ptushki-backend/pull/84 shape of initial data.

Thus it also removes Localize Serivice as only used for getting data from initial data

**There are doubts that some localization will work at all now, because this is the only localization mechanism, and very strange, it had a local context and was used only within tables.**

### How should I test this?
Open observation or rings page and check that descriptions of codes are correctly filled


### Jira ticket related to this PR

- [EPMCUXDA-7721](https://jira.epam.com/jira/browse/EPMCUXDA-7721)

### Link to branch and/or PR's related

- https://github.com/abahachuk/ptushki-backend/pull/84 corresponding PR on backend side

### Type of change

- [ ] Adds new features.
- [x] Changes existing functionality.
- [ ] Deprecates features.
- [ ] Removes features.
- [ ] Fixes any bug.
- [ ] Other: security, build process, infrastructure, tests, docs, etc.
